src/txn/txn_recover.c

```diff
@@ -781,6 +786,7 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
          * earlier time.
          */
         WT_ERR(__recovery_file_scan(&r));
+        log_recovery = true;
```

Comment:
Actually, in this code flow, we don't perform the log recovery. how about the variable name as `no_log_recovery`?

Reply:
Fixed